api/components/TopQuestions.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsxImportSource @hono/hono/jsx */
+import { type FC } from "@hono/hono/jsx";
+import { Question } from "../models/questions.ts";
+import { ArrowUpIcon } from "./assets/arrow-up.ts";
+
+const TopQuestions: FC<{ questions: Question[] }> = ({ questions }) => {
+  // QUESTION:
+  // 1. do we want questions to be scrollable?  currently - `overflow-y: scroll` ;
```

Comment:
> 1. do we want questions to be scrollable?  currently - `overflow-y: scroll` ;

That makes a ton of sense. Let's make it scrollable. 

> 2. What if the length of the question requires line breaks?

Let's have  a line break to not break the layout